pkg/kube/apisix/apis/config/v2beta1/types.go

```diff
@@ -197,3 +197,35 @@ type ApisixRouteList struct {
 	metav1.ListMeta `json:"metadata" yaml:"metadata"`
 	Items           []ApisixRoute `json:"items,omitempty" yaml:"items,omitempty"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
```

Comment:
IMHO it should be a namespaced resource.

Reply:
Yes, we can set it to ns scope, so as to avoid interference